### PR TITLE
API compatible with latest develop branch

### PR DESCRIPTION
**Pull request by KarishmaBothara**
1. Updated docker image to latest develop branch
2. Removed sylo test
3. Modified staking test to use payee from reward storage
4. Added RPC calls for staking(accruedPayout) and grandpa(restartVoter)
5. Updated static metadata in api and types to latest metadata and re-generated types.
6. All packages/types/src/interfaces/augment-* are auto generated

### CODE REVIEW

File: packages/api/test/e2e/staking.e2e.ts [hunk 1]
```diff
@@ -206,6 +206,13 @@ describe('Staking Operations', () => {
     await api.tx.staking.setController(newController.address).signAndSend(stash);
   });
 
+  test('Calculate accruedPayout via RPC', async done => {
+    const rewardDestinationAddress = '5FEe8Ht1ZTzNjQcvrxbLxnykA2EXfqN5LMog2gaNPus4tfZR';
+    const accruedPayout = await api.rpc.staking.accruedPayout(rewardDestinationAddress);
```
Comment (holygits) [inline packages/api/test/e2e/staking.e2e.ts:211]: This should be given a stake account ID

File: packages/api/test/e2e/staking.e2e.ts [hunk 2]
```diff
@@ -207,8 +207,7 @@ describe('Staking Operations', () => {
   });
 
   test('Calculate accruedPayout via RPC', async done => {
-    const rewardDestinationAddress = '5FEe8Ht1ZTzNjQcvrxbLxnykA2EXfqN5LMog2gaNPus4tfZR';
-    const accruedPayout = await api.rpc.staking.accruedPayout(rewardDestinationAddress);
+    const accruedPayout = await api.rpc.staking.accruedPayout(stash.address);
     expect(accruedPayout.toNumber()).toBeGreaterThanOrEqual(0);
     done();
```
Comment (holygits) [inline packages/api/test/e2e/staking.e2e.ts]: there will always be a reward now
```suggestion
    expect(accruedPayout.toNumber()).toBeGreaterThan(0);
    done();
```

Comment (KarishmaBothara) [reply packages/api/test/e2e/staking.e2e.ts]: done..

File: packages/types/src/interfaces/staking/definitions.ts [hunk 3]
```diff
@@ -1,6 +1,18 @@
 // CENNZnet specific staking types, the rest will be resolved from @polkadot/types
 
 export default {
+  rpc: {
+    accruedPayout: {
+      description: 'Retrieves the currently accrued reward for the specified payee (validator or nominator)',
```
Comment (holygits) [inline packages/types/src/interfaces/staking/definitions.ts:6]: Retrieves the currently accrued reward for the specified stash

File: packages/types/src/interfaces/staking/definitions.ts [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+      params: [
+        {
+          payee: 'payee',
+          type: 'AccountId',
```
Comment (holygits) [inline packages/types/src/interfaces/staking/definitions.ts:10]: we might need to fix this on node side also
```suggestion
          stash: 'stash',
          type: 'AccountId',
```

**Review by holygits — APPROVED** [review]

**Pull request merged**